integrations/ga-cross-domain-tracking.md

```diff
@@ -0,0 +1,41 @@
+# Widget Google Analytics 4 Cross-Domain Tracking
+
+> **Notice of usage:** Google Tag Manager (GTM) and Google Analytics 4 (GA4) are third party services and we provide this integration as is.
+We support a set of custom events and Data Layer variables for use with GTM and GA4, however we have no control over what happens to them and how they are used. The basic setup examples we provide have been tested and verified to work with the Mews Booking Engine, however if you need a more complex setup then we cannot provide the support to do so and we recommend to ask a specialist to set it up and test it for you.
+
+## What is cross-domain tracking in Google Analytics 4
+
+It enables webmasters, marketers, and analysts to transfer user or visitor identifiers from domain A to domain B while maintaining session information. As a result, the visitor can be tracked as the same individual across multiple websites or domains.
+
+This is particularly important for widget booking engines. When immediate payment is required, the guest is redirected to a payment gateway on the Mews domain. After the payment, the guest is then redirected back to the booking confirmation screen within the booking engine, but this screen is also on the Mews domain, which differs from the original widget domain.
+
+## What it solves
+
+Google Analytics 4 (GA4) requires cross-domain tracking configuration to accurately track user interactions that span across different domains. Without this configuration, a visitor who completes a booking with a payment through a widget hosted on another domain would be tracked incorrectly. This visitor would appear as two separate users in Google Analytics, resulting in disjointed sessions. Additionally, the referral source attribution would be compromised, potentially misrepresenting the origin of the traffic. Essentially, cross-domain tracking in GA4 ensures that the user journey and source data are correctly linked and reported, even when transactions or interactions occur across multiple domains.
```

Comment:
I edited this section because it is mainly re-stating "what is cross-domain tracking" from the previous section, which is unnecessary, instead of directly answering the question "why is it important?". I think it should say why it is important in the specific context of the Booking Engine integration, hence the part about the user being directed to Mews/payment gateway.